cpp/pybind/visualization/rendering/rendering.cpp

```diff
@@ -119,7 +123,12 @@ void pybind_rendering_classes(py::module &m) {
                     "be accessed after that point.")
             .def("render_to_image", &PyOffscreenRenderer::RenderToImage,
                  "Renders scene to an image, blocking until the image is "
-                 "returned");
+                 "returned")
+            .def("render_to_depth_image",
+                 &PyOffscreenRenderer::RenderToDepthImage,
+                 "Renders scene depth buffer to a float image, blocking until "
+                 "the image is returned. Pixels range from 0 (near plane) to "
+                 "1 (far plane)");
```

Comment:
@prewettg 
Pixels range from 0 (near plane) to 1 (far plane). Then, how do you obtain the actual depth value? Using the near and far plane? It seems there is a missing interface in python for this, or how can you get the near/far plane of the camera (in python)?